packages/bitcore-wallet-service/test/integration/server.js

```diff
@@ -1463,14 +1460,14 @@ describe('Wallet service', function() {
         server.createAddress({}, function(err, address) {
           should.not.exist(err);
           should.exist(address);
-          server.getWallet({}, (err, w) => {
+          server.getWallet({}).then(w => {
 
             var old = server.getWallet;
             server.getWallet = sinon.stub();
 
             // return main address index to 0;
             w.addressManager.receiveAddressIndex = 0;
-            server.getWallet.callsArgWith(1, null, w);
+            server.getWallet.returns(new Promise((resolve) => { resolve(w) }));
```

Comment:
you can use yields instead of new promise

Reply:
I think yield is for functions with callbacks... So I replaced the new promise with `resolves`. Let me know if I am wrong